scripts/update_abi

```diff
@@ -1,20 +1,34 @@
 #!/usr/bin/env bash
 
-#Path to the autonity source code
-AUTONITY_PATH=../../../autonity-internal
 
+# Autonity node tag version
+# The releases are available at: 
+# https://github.com/autonity/autonity/releases
+AUTONITY_NODE_TAG=v0.10.1
+AUTONITY_NODE_COMMIT=ce483153e7c1527c0326e8100b57b102bd702781
+AUTONITY_NODE_REPO=https://github.com/autonity/autonity.git
 
-mkdir -p autonity/abi/temp
-solc --abi -o autonity/abi/temp --overwrite ${AUTONITY_PATH}/autonity/solidity/contracts/Autonity.sol
+echo 
+echo "============================================================"
+echo "Generating ABI files for Autonity contract from version ${AUTONITY_NODE_TAG}"
+echo "Commit: ${AUTONITY_NODE_COMMIT}"
+echo "============================================================"
+
+WORKING_DIR=autonity/abi/temp
```

Comment:
I feel its a better convention to have temp/build directories at the repo top-level (`/temp`) rather than in deep subdirectories, but entirely up to you.